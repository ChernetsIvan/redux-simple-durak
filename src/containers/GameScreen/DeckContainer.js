import { connect } from 'react-redux';
import Deck from './../../components/GameScreen/Deck';

const mapStateToProps = store => ({
    cards: store.game.fullDeck
});

export default connect(mapStateToProps)(Deck);