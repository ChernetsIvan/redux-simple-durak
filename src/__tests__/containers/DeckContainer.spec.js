import React from "react";
import renderer from "react-test-renderer";

import { Provider } from "react-redux";
import { createStore } from "redux";

import DeckContainer from "./../../containers/GameScreen/DeckContainer";
import reducer from "./../../reducers";

describe("DeckContainer snapshot testing", () => {
  let store;

  beforeEach(() => {
    store = createStore(reducer);
  });

  test("Store in initial-state mode", () => {
    const component = renderer.create(
      <Provider store={store}>
        <DeckContainer />
      </Provider>
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
});
