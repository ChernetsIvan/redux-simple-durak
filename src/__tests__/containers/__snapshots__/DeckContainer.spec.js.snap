// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeckContainer snapshot testing Store in initial-state mode 1`] = `
<div
  className="container"
>
  <div
    className="row"
  />
  <div
    className="row"
  >
    <div
      className="col-6"
    />
    <div
      className="col-6"
    >
      <div
        className="container"
      >
        <div
          className="row justify-content-center"
        >
          <div
            className="col-auto"
          >
            <b />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
